File: API/WorkoutTracker.API/Models/ExerciseDTO.cs
using System;

namespace WorkoutTracker.API.Models
{
    //WE NEED BOTH IDS HERE
    public record ExerciseDTO(
        int Id, 
        Guid PublicId, 
        DateTime CreatedDateTime, 
        DateTime? ModifiedDateTime, 
        string Name, 
        string TargetAreas);
}

File: workout-tracker/src/app/workouts/_models/exercise-dto.ts
import { IMightHaveAuditDates } from "../../shared/interfaces/i-might-have-audit-dates";

export class ExerciseDTO implements IMightHaveAuditDates {
  id: number = 0;
  publicId: string = '';
  createdDateTime: Date = new Date();
  modifiedDateTime?: Date | null = null;
  name: string = '';
  targetAreas: string = '';
}
